### DISCUSSION TITLE
Any thoughts about building your firmware selection for MicroPython 1.22 ?

### DISCUSSION THREAD

**Discussion by AndyThirtover** [russhughes/st7789_mpy#153] [General]
Hi Russ

Like many of us I've been enjoying your work for some time ...  what would it take to persuade you to run up some builds for 1.22 ?

Double Regards

Andy

**Comment by russhughes** [comment]
Yes I plan to update the firmware soon.

**Reply by caco3** [reply]
I would suggest to use Github actions for this.
@bcw222 already implemented this for a subset of the supported ESPs: https://github.com/bcw222/st7789_mpy/commit/0008049f527ea8794e2b6c9bc21b3aca4b6ee5e7

**Comment by AndyThirtover** [comment]
I was thinking in terms of a bounty ;) 
Here's a little work I've got going with your stuff ...

Double Regards


Andy

https://github.com/russhughes/st7789_mpy/assets/1166943/29f47577-f325-4327-aa8b-1b9f1e1b4d22

**Comment by asherevan** [comment]
Or preferably even MicroPython 1.25.

**Reply by caco3** [reply]
The mentioned fork already uses Micropython 1.25!
I don't have the display yet, so can't test it.

**Reply by bcw222** [reply]
I have used the ESP32-C3 port on a project, and it worked well.

**Reply by caco3** [reply]
Cool!
Do you want to create a Pull Request to integrate it here? Else I might do it

**Reply by bcw222** [reply]
Maybe you mean this? russhughes/st7789_mpy/pull/183
